test/test_filesystem_helper.cpp

```diff
@@ -229,3 +229,9 @@ TEST(TestFilesystemHelper, remove_extension_no_extension)
   p = rcpputils::fs::remove_extension(p);
   EXPECT_EQ("foo", p.string());
 }
+
+TEST(TestFilesystemHelper, get_cwd)
```

Comment:
can we add here (for example) an env var to check that the current directory returned is the right value?

Reply:
ed673ad